app/controllers/analytics_controller.rb

```diff
@@ -10,8 +9,14 @@ class AnalyticsController < Sellers::BaseController
   def index
     authorize :analytics
 
+    @hide_nav = true
     @analytics_props = AnalyticsPresenter.new(seller: current_seller).page_props
     LargeSeller.create_if_warranted(current_seller)
+
+    render inertia: "Analytics/index",
+           props: RenderingExtension.custom_context(view_context).merge(
```

Comment:
Can you move `RenderingExtension.custom_context(view_context).merge` to a method `inertia_props` within `ApplicationController` or a module that it would include like `InertiaRendering`:

```ruby
def inertia_props(props)
  RenderingExtension.custom_context(view_context).merge(props)
end
```

This would clean up the repetitive pattern in controllers like `balance_controller.rb`, `dashboard_controller.rb`, etc.

Reply:
done

Reply:
You forgot to use `inertia_props` in this file